utils/release.yml

```diff
@@ -29,13 +29,13 @@ notes:
       regexp: '(packaging|pkg|rpm|deb|pkg-config|configure|config|[Mm]ake|build): '
 
     - title: Docker
-      regexp: '[Dd]ocker(/[^ ]+)?: '
+      regexp: '[Dd]ocker(/[^ ]+)?(\(\w[\w.]*\))?: '
```

Comment:
Sync the scope with the regex of the CI category

Reply:
Fixed, also for Singularity.